src/style.css

```diff
@@ -1 +1,90 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+body {
+  margin: 0;
+}
+
+.head {
+  width: 100%;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-size: 12px;
+}
+
+.nav {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: rgb(255, 255, 255);
+  min-width: 1024px;
+  max-width: 1200px;
+  margin: 0 auto;
+}
+
+.nav_link {
+  display: flex;
+  text-transform: uppercase;
+  padding: 0;
+  margin: 0;
+  margin-right: 50px;
+}
+
+.logo-link {
+  margin: 10px;
+  margin-right: 0;
+  margin-left: 50px;
+  height: 40px;
+}
+
+.nav_link :first-child {
+  margin: 0 10px 0 0;
+}
+
+.nav_link :last-child {
+  margin: 0 0 0 10px;
+}
+
+.nav_link > :nth-child(1n+2):not(:last-child) {
+  margin: 0 10px;
+}
+
+.refer:link {
+  text-decoration: none;
+  color: black;
+  position: relative;
+  padding: 22.5px 0;
+}
+
+.blue:link {
+  color: #00acdc;
+}
+
+.refer:visited {
+  color: #00acdc;
+}
+
+.refer:hover {
+  color: #00acdc;
+}
+
+.is-active:active::after {
+  display: block;
+  position: absolute;
+  bottom: 0;
+  content: "";
+  width: 100%;
+  height: 4px;
+  background: #00acdc;
+  border-radius: 8px;
+}
+
+.nav_link :first-child::after {
+  display: block;
+  position: absolute;
+  bottom: 0;
+  content: "";
+  width: 100%;
+  height: 4px;
+  background: #00acdc;
+  border-radius: 8px;
+}
```

Comment:
why have you added two blocks with the same styles?

Reply:
because all links in state "active" should be with line bellow them, and link "apple" should be with line in default state also... so, when i wrote the text above,  i have realized... ) i can write like this : .is-active:active::after {
  display: block;
  position: absolute;
  bottom: 0;
  content: "";
  width: 100%;
  height: 4px;
  background: #00acdc;
  border-radius: 8px;
}